ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -183,6 +183,13 @@ export default class TaskRunsCard extends Component {
   };
 
   handleToggleArtifacts = () => {
+    const { history } = this.props;
+    const { taskId } = this.getCurrentRun();
+    const { showArtifacts } = this.state;
+
+    !showArtifacts
+      ? history.replace(`/tasks/${taskId}#artifacts`)
+      : history.replace(`/tasks/${taskId}`);
```

Comment:
If a user navigates to https://community-tc.services.mozilla.com/tasks/QW3amtG3SkK3Tp5lBSq_tg#artifacts, will this have the artifacts panel open on page load?

Reply:
No, how should we deal with that?

Reply:
You can do this by having `showArtifacts` linked to the hash instead of state.

Instead of 

```
const { showArtifacts } = this.state;
```

We could probably just do:

```
const showArtifacts = window.location.hash === '#artifacts';
```